onos-config/values.yaml

```diff
@@ -23,7 +23,7 @@ replicaCount: 1
 image:
   registry: ""
   repository: onosproject/onos-config
-  tag: v0.10.10
+  tag: v0.10.11
```

Comment:
It seems tag an app version doesn't match

Reply:
changed the chart so that the `.Chart.AppVersion` is used as the image tag. No need to duplicate the value (otherwise I'll always forget to update :) )

Reply:
yeah we should do that.